clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -280,6 +280,13 @@
             " be set to `false` when using brokers older than 0.11.0";
     public static final boolean DEFAULT_ALLOW_AUTO_CREATE_TOPICS = false;
 
+    /** <code>skip.metadata.cache.update.upon.unassign</code> */
+    public static final String SKIP_METADATA_CACHE_UPDATE_UPON_UNASSIGN = "skip.metadata.cache.update.upon.unassign";
```

Comment:
since this is a hotfix could you start this key with "linkedin"? (so "linkedin.skip.metadata .....")
this ensures we ~never conflict with anything new in upstream

Reply:
Good point. Added